### ISSUE TITLE
Automatically create accounts for staff

### ISSUE THREAD

**Issue by jornane** [osTicket/osTicket-plugins#37]
I use the passthru plugin in combination with mod_mellon. Only staff can log in through mod_mellon. I would like it if staff logging in this way will be automatically created in osTicket when they are not in the database yet. Is there a way to do this, or can this be included as a configuration option to the auth plugins?

**Comment by greezybacon** [comment]
Never heard of mod_mellon. Glad to hear of the creative thinking.

At the time, we have no plans to auto-create staff members

**Comment by wanno-drijfhout** [comment]
The one true lacking of osTicket seems its limited integration with other IT systems. We already have a customer database and a user account management system, as most enterprises do. I would very much appreciate it if osTicket would support the proper functionalities, or at least the hooks to develop our own.

**Comment by DieQuelle** [comment]
Is it also not possible atm that osticket creates client accounts automatically on correct login credentials from MS AD?

My Authentication is working but why should I import them every few days when new people join our company?

**Comment by ntozier** [comment]
only Agents need accounts?

**Comment by DieQuelle** [comment]
Then our users need to insert there email address everytime.
I think the solution is to sync all users of our AD with email and password to the sql database of osticket?

Is this correct?

**Comment by Chefkeks** [comment]
@DieQuelle 
It is possible, just combine the ldap and the http pass thru plugin ;)
We use it that way in my company and there is no need to create any user account anymore. Beside we created an unofficial add on (PHP scripts) to update information like Phone numbers, Office, etc. via a cron job from MS AD to osTicket. Just see my repository here at github ;)

If you need more details, you can open a discussion in the osTicket forum. osTicket.com/forum

**Comment by Richiricheh** [comment]
> @DieQuelle
> It is possible, just combine the ldap and the http pass thru plugin ;)
> We use it that way in my company and there is no need to create any user account anymore. Beside we created an unofficial add on (PHP scripts) to update information like Phone numbers, Office, etc. via a cron job from MS AD to osTicket. Just see my repository here at github ;)
> 
> If you need more details, you can open a discussion in the osTicket forum. osTicket.com/forum

I agree with this. There is also a means of simply unpacking the LDAP PHAR and adding in additional fields to sync and pull into osTicket.
